QuizApp/QuizApp/User/UserViewModel.swift

```diff
@@ -16,31 +17,50 @@ class UserViewModel {
         getUserInfo()
     }
 
-    func save(username: String) {
-        userUseCase.save(userInfo: UserInfoModel(username: username))
+    func save(username: String, name: String) async {
+        do {
+            try await userUseCase.save(userInfo: UserInfoModel(username: username, name: name))
```

Comment:
After you modify the name, you should reload the account data. However, I saw that you do not return the account data from the network response in this call. Why? You can use that.